src/Plugin/views/argument_default/EntityRenderHistory.php

```diff
@@ -85,6 +98,14 @@ public function getArgument() {
     $ids = $this->getEntityRenderHistory()
       ->getRenderedEntities($this->options['entity_type_id']);
 
+    if (!$ids) {
```

Comment:
I think we would have to merge the ids here.

Reply:
done